crypto/CMakeLists.txt

```diff
@@ -460,11 +461,6 @@ add_library(
   x509/x_val.c
   x509/x_x509.c
   x509/x_x509a.c
-  x509v3/pcy_cache.c
```

Comment:
You have the change to remove pcy_cache.c but in the merge conflict you didn't actually delete the file.

Reply:
fixed